cczoo/common/docker/gramine/anolisos-gramine-sgx-dev.dockerfile

```diff
@@ -0,0 +1,99 @@
+
+# Copyright (c) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://github.com/oscarlab/graphene/blob/master/Tools/gsc/images/graphene_aks.latest.dockerfile
+FROM openanolis/anolisos:8.4-x86_64 AS Anolisos
+
+ENV INSTALL_PREFIX=/usr/local
+ENV LD_LIBRARY_PATH=${INSTALL_PREFIX}/lib:${INSTALL_PREFIX}/lib64:${LD_LIBRARY_PATH}
+ENV PATH=${INSTALL_PREFIX}/bin:${LD_LIBRARY_PATH}:${PATH}
+# Add steps here to set up dependencies
+RUN yum install -y \
+    openssl-devel \
+    libcurl-devel \
+    protobuf-devel \
+    yum-utils.noarch \
+    python3 \
+    wget
+
+# Intel SGX
+RUN mkdir /opt/intel && cd /opt/intel \
```

Comment:
Please combine these commands and remove the downloaded images after the installation.